module.txt

```diff
@@ -1,7 +1,7 @@
 {
     "id" : "Potions",
     "version" : "0.2.1-SNAPSHOT",
-    "author" : "bi0hax, XTariq, shardulc, S-ulphuric, kallentu, Mandar Juvekar, Arpit Kamboj (digitalripperynr), SufurElite, JayGupta2003, He Who Shall Not Be Named (VaibhavBajaj)",
+    "author" : "bi0hax, XTariq, shardulc, S-ulphuric, kallentu, Mandar Juvekar, Arpit Kamboj (digitalripperynr), SufurElite, JayGupta2003, He Who Shall Not Be Named (VaibhavBajaj)", FieryPhoenix909, 
```

Comment:
take care of the json format, it is `"key" : "value"`. The module will not load if the format is not parseable.
The correct ending of the line would be
`(VaibhavBajaj), FieryPhoenix909",`
instead of
`(VaibhavBajaj)", FieryPhoenix909, `